modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/DriverInfo/DriverInfoForm.tsx

```diff
@@ -553,59 +599,103 @@ const DriverInfoForm = ({
 						</ClaySelect>
 
 						<label htmlFor="occupation">
-							Occupation&nbsp;
+							Ocuppation&nbsp;
 							<span className="text-danger-darken-1">*</span>
 						</label>
+
+						{hasError.occupation && <ErrorMessage />}
 					</div>
 
-					<div className="col filled form-condensed form-group">
+					<div
+						className={classNames(
+							'col filled form-condensed form-group',
+							{
+								'has-error': hasError.otherOccupation,
+							}
+						)}
+					>
 						<ClayInput
 							autoComplete="off"
-							className="bg-neutral-0 otherOcupation w-100"
-							disabled={occupationValue !== 'other'}
-							id="otherOcupation"
-							name="otherOcupation"
+							className="bg-neutral-0 otherOccupation w-100"
+							disabled={occupationValue !== 'Other'}
+							id="otherOccupation"
+							name="otherOccupation"
+							onBlur={(event) => {
+								setHasError({
+									...hasError,
+									otherOccupation:
+										event.target.value === ''
+											? true
+											: false,
+								});
+							}}
 							onChange={(event) => {
-								handleChangeField(
-									'otherOcupation',
-									event.target.value,
-									id
-								);
+								let value = event.target.value;
+								value =
+									value === 'Choose an option' ? '' : value;
+								handleChangeField('otherOccupation', value, id);
+								setHasError({
+									...hasError,
+									otherOccupation:
+										value === '' &&
+										occupationValue === 'other'
+											? true
+											: false,
+								});
 							}}
 							required
 							type="text"
+							value={form[formNumber - 1].otherOccupation}
 						/>
 
-						{occupationValue === 'other' && (
+						{occupationValue === 'Other' && (
 							<label htmlFor="otherOccupation">
-								Other Occupation
+								Other Ocupation
 								<span className="text-danger-darken-1">*</span>
 							</label>
 						)}
 
-						{occupationValue !== 'other' && (
+						{occupationValue !== 'Other' && (
 							<label htmlFor="otherOccupation">
-								Other Occupation
+								Other Ocupation
```

Comment:
Please add back the 'c'
```suggestion
								Other Occupation
```